### DISCUSSION TITLE
PR List: what to do with the PR that were open before applying GitFlow?

### DISCUSSION THREAD

**Discussion by costero-e** [ga4gh-beacon/beacon-v2#66] [General]
Maybe the ones that have worked in mostly can better make a conclusion on what to do with each of them, I will then do the actions decided making PR towards develop branch. 
As for PR #61 it seems work has been done and could be PRed, then #58 seems to not having an agreement on the PR and the others:  #47 #52 #53 #54 seem to need a review?
All of them need to be merged first to their parent feature branch and then PRed to develop, not main.

**Comment by mbaudis** [comment]
+1 for #61

**Reply by costero-e** [reply]
#61 has been merged into resultsets branch and will be PRed to develop when we finish deciding if we discard #52

**Comment by mbaudis** [comment]
Well, I haven't seen any comment on #54 - so maybe I've missed something there but from my perspective still +1.

**Reply by costero-e** [reply]
#54 has been replaced by #67 and merged into clean-up feature branch, which will be PRed into develop when we are ready with clean-up feature branch

**Comment by mbaudis** [comment]
IMO #52 should be followed through (possibly w/ adjustments, definitely w/ documentation) in favour of #65. For me the `booleanResultSets` and `countResultSets` ... option there would in principle work but have the disadvantage of limiting to a single style of response (instead of setting just the maximum/wished for granularity).

**Comment by mbaudis** [comment]
Closing since branch stuff has moved on...
